src/poetry/console/commands/lock.py

```diff
@@ -37,11 +37,13 @@ def handle(self) -> int:
         )
 
         if self.option("check"):
-            return (
-                0
-                if self.poetry.locker.is_locked() and self.poetry.locker.is_fresh()
-                else 1
+            if self.poetry.locker.is_locked() and self.poetry.locker.is_fresh():
+                self.line("poetry.lock is up to date")
+                return 0
+            self.line(
+                "Error: poetry.lock is out of date. Run `poetry update` to fix it"
```

Comment:
You should use `<error>...</error>` as in https://github.com/python-poetry/poetry/blob/28d05d002475f0cc55e7cc46ae01f9a3ce4e03a8/src/poetry/console/commands/check.py#L26
for coloring.

Reply:
Should we wait on an opinion from traiate team on this one?

Reply:
I think adding `<error>...</error>` is safe.

Considering the wording: Since there are no replies yet, either nobody had time to think about it or has no strong opinion or is just fine with the proposal. 😉 If you want to speed up the process (and are not afraid from some more iterations of changes) we can push this PR until I will approve it. Before merging, a core member will look at the changes anyway and may (or may not) request further changes.

Thus, if you want to proceed, I propose to stick with the wording from the documentation (`poetry.lock` is (not) consistent with `pyproject.toml`) and recommend running `poetry lock [--no-update]`. Otherwise, you can also just wait some more days for additional opinions.

Reply:
> I propose to stick with the wording from the documentation (poetry.lock is (not) consistent with pyproject.toml) and recommend running poetry lock [--no-update]


I like this one :+1:

Reply:
Updated messages :)